### DISCUSSION TITLE
Welcome to BlockSuite Discussions!

### DISCUSSION THREAD

**Discussion by doodlewind** [toeverything/blocksuite#31] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by thedigitalworkplace** [comment]
Hi,

How can I get the template available in Affine? I managed to deployed the blocksuite, but interested if can use the template available in Afffine https://github.com/toeverything/affine
